### ISSUE TITLE
‘This version’ link is broken

### ISSUE THREAD

**Issue by andreastt** [w3c/webdriver#1342]
The __This version__ link at the top of the [living document](https://w3c.github.io/webdriver/) published here on GitHub appears to be broken. The link is https://www.w3.org/TR/2018/LD-webdriver-20181102/.

I understand this is some sort of living document (LD) snapshot stored under /TR, but I don’t understand the process under which it gets published there. Perhaps @sideshowbarker can shed some light on this.

**Comment by andreastt** [comment]
Same goes for the __Previous version__ link.

**Comment by sideshowbarker** [comment]
This needs to be fixed in the respec sources I guess. For one thing, the URL should not have `LD` in it. It’s not an abbreviation used in W3C publishing. It should instead be `WD` —  https://www.w3.org/TR/2018/WD-webdriver-20181102/.

I thought that my patch to give respec Living-Document support would actually cause it to generate URLs with `WD` — and when I tested locally, I seem to recall that it had. But I guess I was mistaken.

But really, respec rightly should also not be adding **This version** links to LD/LS documents at all (or to EDs; I can’t recall now if it does for EDs).

So, bottom line is that it seems like there are some further fixes needed in respec to make it do the right things for LD/LS documents. I hope somebody other than me can make time to do those fixes. But if nobody else does within the next couple weeks, I guess I can try to make time for it myself.
